Education_Pathways/frontend/src/components/Navbar.js

```diff
@@ -116,9 +102,6 @@ We are looking for feedback to improve Education Pathways and make it more usefu
             <Route path="/review">
             <div>
               <FeedbackSubmitPage/>
-              <section>
```

Comment:
Test to demonstrate the feedback item. Can be deleted now.

Reply:
Reviewed. No issues observed